gcp/secure_swarm_node/main.tf

```diff
@@ -24,6 +24,15 @@ resource time_static self {
   }
 }
 
+//Updates the google_compute_disk_resource_policy_attachment when google_compute_resource_policy is updated
+resource time_static snapshot {
+  triggers = {
+    snapshot_properties = jsonencode(var.snapshot_properties)
+    daily_schedule = jsonencode(var.data_disk_snapshot_schedule)
```

Comment:
Change the name of this.

Reply:
Changed to `resource_policy_time` (`time_static.self` already exists)